raaghu-elements/src/rds-card/rds-card.stories.tsx

```
@@ -591,9 +642,18 @@
 
+const cardWithLineChartArgs = {
+    state: "Default",
+    style: "Default",
+    showIndicator: false,
+    colorVariant: "primary",
+    borderColor: "",
+    isDisabled: false,
+};
+
 export const CardWithLineChart: Story = {
   args: {
```

ESLint

[error] Expected indentation of 4 spaces but found 2.

(indent)